src/Microsoft.DotNet.Interactive.Formatting.Tests/TabularDataFormatterTests.cs

```diff
@@ -27,9 +27,9 @@ public void can_generate_tabular_json_from_object_array()
             {
                 new { Name = "Q", IsValid = false, Cost = 10.0 },
                 new { Name = "U", IsValid = false, Cost = 5.0 },
-                new { Name = "E", IsValid = true, Cost = 10.0 },
-                new { Name = "S", IsValid = false, Cost = 10.0 },
-                new { Name = "T", IsValid = false, Cost = 10.0 }
+                new { Name = "E", IsValid = true, Cost = 10.2 },
```

Comment:
Why this change?

Reply:
to ensure number serialization preserves decimals, the new serialization seems to be too smart about decimals with 0 value after the period